cypress/integration/lockedOutUser.js

```diff
@@ -0,0 +1,24 @@
+import LockedOutUserPage from './../support/PageObjects/LockedOutUserPage'
+
+    // Creating Object for LockedOutUserPage
+    const lockedOutPage = new LockedOutUserPage();
+
+    describe('Locked Out User', function () {
+        before(function () {
+            const lockedLogin = ['locked_out_user'];
```

Comment:
it's better to keep this data in env variables: https://docs.cypress.io/guides/guides/environment-variables#Setting

Reply:
Correct if I'm wrong but it is not env variables. 
`lockedLogin` and `correctPassword` are credentials.